live: Add manifest ID to openapi and logs

j0sh opened the pull request:
Helps filter logs by orchestrator once swapping is in.

leszko commented on runner/gateway.openapi.yaml:
```diff
@@ -878,11 +876,9 @@ components:
         message:
-          allOf:
-          - $ref: '#/components/schemas/LLMMessage'
+          $ref: '#/components/schemas/LLMMessage'
```
I think you need to revert this OpenAPI changes from `allOf`, because the CI is failing. Maybe you're using some different lib version for generating OpenAPI.

j0sh replied:
Ah yeah this is an [older syntax](https://stackoverflow.com/a/73490368). Just tried a bunch of things to get CI to pick up the newer syntax (updated fastapi, updated python) but nothing seemed to work so I just manually picked the changes in https://github.com/livepeer/ai-runner/pull/579/commits/0aaed81f4ade646083dd2ef491b028c36357fd9a

> In OpenAPI 3.1 (which uses JSON Schema 2020-12 by default), there's even no need for allOf if you only have one $ref because $ref now [allows sibling keywords](https://stackoverflow.com/a/41752575/113116).

leszko reviewed (approved):
Added one comment, other than that LGTM

j0sh commented:
Holding off on merging this until after https://github.com/livepeer/ai-runner/pull/568

j0sh commented:
Rebased onto master and fixed merge conflicts, no functional changes.

Had to update the OpenAPI yamls to make CI happy since they weren't regenerated in  https://github.com/livepeer/ai-runner/pull/568 . This also probably needs a go-livepeer update (although I suspect it is not a functional change) - https://github.com/livepeer/go-livepeer/pull/3583
